tests/zlib/ref.txt

```diff
@@ -1,4 +1,4 @@
-zlib version 1.2.5 = 4688, compile flags = 85
```

Comment:
Do you know what this change is?

Reply:
Yeah "compile flags" encodes a bitfield for the size of off_t